### PR TITLE
Recomender Infrastructure Template Code

### PR DESCRIPTION
**Pull request by mannyrivera2010**
Please Review
@lsemesky 
@J-Fid 
@seenusuvarna 
@skhtet

### CODE REVIEW

**Comment by akonsowski** [comment]
:+1:

**Comment by seenusuvarna** [comment]
:+1:

File: ozpcenter/recommend/recommend.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    recommender_result_set: Dictionary with profile id, nested listing id with score pairs
+        {
+            profile_id#1: {
+                listing_id#1: score#1,
+                listing_id#2: score#2
+            },
+            profile_id#2: {
+                listing_id#1: score#1,
+                listing_id#2: score#2,
+                listing_id#3: score#2,
```
Comment (J-Fid) [inline ozpcenter/recommend/recommend.py]: Should this say score#3?

Comment (mannyrivera2010) [inline ozpcenter/recommend/recommend.py]: It does not have to

Comment (J-Fid) [inline ozpcenter/recommend/recommend.py]: Oh, ok.

**Comment by J-Fid** [comment]
:+1:

**Pull request merged**